memcached.c

```diff
@@ -4455,7 +4455,7 @@ static int _mc_meta_save_cb(const char *tag, void *ctx, void *data) {
     {
         struct timeval tv;
         gettimeofday(&tv, NULL);
-        restart_set_kv(ctx, "stop_time", "%lu", tv.tv_sec);
+        restart_set_kv(ctx, "stop_time", "%llu", (unsigned long long) tv.tv_sec);
```

Comment:
For reference, I've updated this to match the similar typecast just a few lines up (one line outside the default GitHub context):

```c
    restart_set_kv(ctx, "process_started", "%llu", (unsigned long long) process_started);
```